paddle/operators/activation_op.cc

```diff
@@ -338,6 +338,37 @@ class ThresholdedReluOpMaker : public framework::OpProtoAndCheckerMaker {
   }
 };
 
+template <typename AttrType>
+class HardSigmoidOpMaker : public framework::OpProtoAndCheckerMaker {
+ public:
+  HardSigmoidOpMaker(framework::OpProto *proto,
+                     framework::OpAttrChecker *op_checker)
+      : OpProtoAndCheckerMaker(proto, op_checker) {
+    AddInput("X", "Input of HardSigmoid operator");
+    AddOutput("Y", "Output of HardSigmoid operator");
+    AddComment(R"DOC(
+Hard Sigmoid activation operator.
+
+Segment-wise linear approximation of sigmoid[1].
+This is much faster than sigmoid.
+
+hard_sigmoid = max(0, min(1, slope * x + shift))
+
```

Comment:
Is it true that both slope and shift are non-negative? If yes, it will be helpful to mention as a comment.

Reply:
Fixed in f4820c7dd10f54eb3772a4abc5b3f408072cbbae

Reply:
Corrected comment grammatical mistake in 7b1054676248a12ec536d36a0ffab34b29c0798b